source/1_engine/rex_engine/src/string/stringpool.cpp

```diff
@@ -0,0 +1,155 @@
+#include "rex_engine/string/stringpool.h"
+
+#include "rex_engine/diagnostics/assert.h"
+#include "rex_engine/string/stringentry.h"
+#include "rex_std/functional.h"
+#include "rex_std/internal/utility/pair.h"
+#include "rex_std/unordered_map.h"
+
+namespace rsl
+{
+  inline namespace v1
+  {
+    // https://en.wikipedia.org/wiki/Fowler–Noll–Vo_hash_function
+    rsl::hash_result hash_bytes(const void* data, rsl::count_t size) noexcept
```

Comment:
we already have a hash implementation for bytes.
probably best if we stick with that so all string hashes are equivalenet to each other

Reply:
Could you tell me where I can find it?